src/Sdk/Sdk.csproj

```diff
@@ -13,6 +13,10 @@
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     </PropertyGroup>
 
+    <ItemGroup>
+        <InternalsVisibleTo Include="Test" />
```

Comment:
I am trying to remember why I needed this. I think it was for a template token extension method to assert the type